src/perf.ts

```diff
@@ -0,0 +1,279 @@
+/** Library used for measuring performance. */
```

Comment:
Could you document how to use this here? E.g, an example of all the lines you need to add to start monitoring an ex command.

It's reasonably obvious when one looks at the whole commit but I'd rather that we didn't have to hunt for it.

Cheers.